_posts/2025-08-03-columna-firmantes.md

```
@@ -0,0 +1,18 @@
+gh-repo: /capitalexpress/capitalexpress.github.io
+gh-badge: [star, fork, follow]
+tags: [Riesgo, Ejecutivo, Solicitud, Firmas]
+comments: false
+author: Catalina Pinilla
+---
+
+## Solicitud de línea de crédito - Firmas.
+
+Cuando se requiere aprobar una solicitud, será más fácil identificar a los usuarios que ya han firmado, sin la necesidad de ingresar en cada uno de las solicitudes de línea de crédito para poder obtener esta información, ahora se ha habilitado la columna firmantes donde podrás ver el nombre de aquellos que ya dieron su firma.
+
+#### Aquí te mostramos como se visualiza esta nueva implementación.
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)